### PR TITLE
Fix perf test due to fortio build error.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Fix perf test due to fortio build error.

## Issue reference

Please reference the issue this PR will close: N/A

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by yaron2 — APPROVED** [review]

File: tests/apps/perf/tester/Dockerfile [hunk 1]
```diff
@@ -10,11 +10,11 @@ FROM golang:1.15 as fortio_build_env
 WORKDIR /fortio
 ADD "https://api.github.com/repos/fortio/fortio/branches/master" skipcache
```
Comment (daixiang0) [inline tests/apps/perf/tester/Dockerfile:11]: What is it for? Add string to file?

Comment (artursouza) [reply tests/apps/perf/tester/Dockerfile:11]: To force a cache miss so the code below is always executed in local development.

**Pull request merged**